examples/jsm/loaders/MaterialXLoader.js

```diff
@@ -44,6 +44,37 @@ const mx_power = ( in1, in2 = float( 1 ) ) => pow( in1, in2 );
 const mx_atan2 = ( in1 = float( 0 ), in2 = float( 1 ) ) => atan2( in1, in2 );
 const mx_timer = () => timerLocal();
 const mx_frame = () => frameId;
+const mx_invert = ( in1, amount = null ) => {
```

Comment:
Could you use the approach below instead of checking node values? Type conversions are done automatically prioritizing large vectors, if the input of `in1` is `vec3` it will return a `vec3`, and so we'll assume that `amount` can be classified as a node abstraction not just for constants.

```js
const mx_invert = ( in1, amount = float( 1 ) ) => sub( amount, in1 );
```

Reply:
Thanks, that's much better! Updated.